### PR TITLE
Add docs for Inspirations support

### PR DESCRIPTION
**Pull request by KnightMiner**
Added docs for jaredlll08/ModTweaker#610

### CODE REVIEW

File: docs/Mods/Modtweaker/Inspirations/Handlers/Cauldron.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+Inspirations features three types of recipes using fluids: ones to transform an item using a fluid, ones to transform a fluid using an item, and ones to add fill with a fluid using an item. While all the handlers take on a fluid stack, only the fluid type will affect the recipe. Inspirations cauldrons use the vanilla system of three bottles in a cauldron so amounts are handled in another parameter.
+
+### Fluid adding
+
+Adds a recipe converting the input item to the output item using the fluid.
+
+* Input item supports stack sizes to require the held stack to be a certain size.
+* Levels determines how many levels are consumed by the recipe. Supports 0-3, defaulting to 1
+* Boiling determines if the cauldron must be placed above fire for the recipe. Can be true to require it, false to require not having fire, or null (default) to ignore it.
```
Comment (kindlich) [inline docs/Mods/Modtweaker/Inspirations/Handlers/Cauldron.md]: That is wrong.
A boolean can never be `null`!
Default is false.

File: docs/Mods/Modtweaker/Inspirations/Handlers/Cauldron.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+mods.inspirations.Cauldron.removeFluidRecipe(<minecraft:beetroot_soup>);
+```
+
+### Fluid transform adding
+
+Adds a recipe converting the fluid to the output fluid using the item.
+
+* Input item supports stack sizes to require the held stack to be a certain size.
+* Max level determines the maximum amount of fluid allowed for this transformation to happen. Used to allow recipes to have a cheaper version if the cauldron contains less fluid.
+* Boiling determines if the cauldron must be placed above fire for the recipe. Can be true to require it, false to require not having fire, or null (default) to ignore it.
```
Comment (kindlich) [inline docs/Mods/Modtweaker/Inspirations/Handlers/Cauldron.md]: That is wrong.
A boolean can never be `null`!
Default is false.

File: docs/Mods/Modtweaker/Inspirations/Handlers/Cauldron.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+mods.inspirations.Cauldron.removeBrewingRecipe("inspirations:haste");
+mods.inspirations.Cauldron.removeBrewingRecipe("minecraft:awkward", "minecraft:water", <minecraft:nether_wart>);
+```
+
+### Potion recipe adding
+
+Adds a recipe converting the input item to the output item using the potion.
+
+* Levels determines how many levels are consumed by the recipe. Supports 0-3, defaulting to 1
+* Boiling determines if the cauldron must be placed above fire for the recipe. Can be true to require it, false to require not having fire, or null (default) to ignore it.
```
Comment (kindlich) [inline docs/Mods/Modtweaker/Inspirations/Handlers/Cauldron.md]: That is wrong.
A boolean can never be `null`!
Default is false.

**Comment by KnightMiner** [comment]
Updated mkdocs

File: docs/Mods/Modtweaker/Inspirations/Inspirations.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# Inspirations
```
Comment (kindlich) [inline docs/Mods/Modtweaker/Inspirations/Inspirations.md]: You did not add this file to mkdocs.yml, so you will not be able to access it in RTD, if you dont need it, remove it, if you do add it to mkdocs.yml

File: mkdocs.yml [hunk 5]
```diff
@@ -366,6 +366,10 @@ pages:
         - Squeezer: 'Mods/Modtweaker/Forestry/Squeezer.md'
         - Still: 'Mods/Modtweaker/Forestry/Still.md'
         - Thermionic Fabricator: 'Mods/Modtweaker/Forestry/Thermionic_Fabricator.md'
+      - Inspirations:
+        - Inspirations: 'Mods/Modtweaker/Inspirations/Crusher.md'
```
Comment (kindlich) [inline mkdocs.yml]: This file does not exist.

Comment (KnightMiner) [reply mkdocs.yml]: How did that get in there, must have not saved properly.

Comment (kindlich) [reply mkdocs.yml]: You probably simply mistook `Crusher.md` for `Inspirations.md`

**Comment by KnightMiner** [comment]
Fixed the link to the main page for Inspirations

**Pull request merged**